tide/core/data/FFMPEGVideoStream.cpp

```diff
@@ -209,7 +202,9 @@ bool FFMPEGVideoStream::isStereo() const
 
 double FFMPEGVideoStream::getDuration() const
 {
-    return std::max(_frameDurationInSeconds * (_numFrames - 1), 0.0);
+    return (_numFrames > 1)
```

Comment:
Since all numbers are signed you do not need to check _numFrames > 1.

Reply:
It's not obvious, but the problem is slightly different.
When _numFrames == 1 then _frameDurationInSeconds is "inf". This gives for the operation:
"inf" * 0 => return "nan"
But we want the returned duration to be 0.
Does that make sense?

Reply:
Okay, seems like your input is absurd then. If you have one frame then I assume that frame is the length of the whole movie.

Reply:
I'm not sure what is really correct in this weird corner case. You think it's better to set _frameDurationInSeconds=0 if there's only a single frame and remove the change here?

Reply:
I can give this a try

Reply:
works this way too, commit updated